pkgs/os-specific/linux/ffado/default.nix

```diff
@@ -41,43 +45,32 @@ mkDerivation rec {
 
   src = fetchurl {
     url = "http://www.ffado.org/files/libffado-${version}.tgz";
-    hash = "sha256-0iFXYyGctOoHCdc232Ud80/wV81tiS7ItiS0uLKyq2Y=";
+    hash = "sha256-xELFL60Ryv1VE7tOhGyFHxAchIT4karFRe0ZDo/U0Q8=";
   };
 
   prePatch = ''
     substituteInPlace ./support/tools/ffado-diag.in \
       --replace /lib/modules/ "/run/booted-system/kernel-modules/lib/modules/"
   '';
 
-  patches = [
-    # fix installing metainfo file
-    ./fix-build.patch
-
-    (fetchpatch {
-      name = "musl.patch";
-      url = "http://subversion.ffado.org/changeset?format=diff&new=2846&old=2845";
-      stripLen = 2;
-      hash = "sha256-iWeYnb5J69Uvo1lftc7MWg7WrLa+CGZyOwJPOe8/PKg=";
-    })
-  ];
-
-  nativeBuildInputs = [
-    desktop-file-utils
-    (scons.override { python3Packages = python311.pkgs; })
-    pkg-config
-    which
-    python
-    python.pkgs.pyqt5
-    wrapQtAppsHook
-  ];
+  nativeBuildInputs =
+    [
+      (scons.override { python3Packages = python311.pkgs; })
```

Comment:
Do we still need the scons pin?

Reply:
Yes, the `SConstruct` still pulls in `import distutils.sysconfig`, which was removed in Python 3.12.

Reply:
Cool. Can you add a comment for that so whoever updates this next knows to remove it?